packages/babel-plugin-transform-classes/src/inline-createSuper-helpers.js

```diff
@@ -0,0 +1,71 @@
+import { template, types as t } from "@babel/core";
+
+const helperIDs = new WeakMap();
+
+export default function addCreateSuperHelper(file) {
+  try {
+    return file.addHelper("createSuper");
```

Comment:
I tested this PR by commenting out this line: all the `exec` tests passed, and the new `output`s made sense. I suggest trying it locally.

If you have an idea on how to better test this, please tell me!